File: src/main/java/top/zbeboy/isy/web/bean/data/schoolroom/SchoolroomBean.java
package top.zbeboy.isy.web.bean.data.schoolroom;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.Schoolroom;

/**
 * Created by zbeboy on 2017/5/31.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class SchoolroomBean extends Schoolroom {
    private String schoolName;
    private String collegeName;
    private String buildingName;
    private int schoolId;
    private int collegeId;
}

File: src/main/java/top/zbeboy/isy/web/bean/data/schoolroom/SchoolroomBean.kt
package top.zbeboy.isy.web.bean.data.schoolroom

import top.zbeboy.isy.domain.tables.pojos.Schoolroom

/**
 * Created by zbeboy 2017-12-08 .
 **/
class SchoolroomBean: Schoolroom() {
    var schoolName: String? = null
    var collegeName: String? = null
    var buildingName: String? = null
    var schoolId: Int = 0
    var collegeId: Int = 0
}